src/biokbase/narrative/viewers.py

```diff
@@ -43,7 +40,7 @@ def view_as_clustergrammer(
 
     generic_df = get_df(ws_ref, col_categories, row_categories, True)
 
-    net = Network(clustergrammer_widget)
+    net = Network(CGM2)
```

Comment:
Updating the widget is kind of an anticlimax!

Reply:
Better than rewriting everything!